Source/WebCore/css/html.css

```diff
@@ -496,10 +496,15 @@ input:is([type="date"], [type="time"], [type="datetime-local"], [type="month"],
     overflow: hidden;
 #if defined(WTF_PLATFORM_IOS_FAMILY) && WTF_PLATFORM_IOS_FAMILY
     border: 1px solid -webkit-control-background;
-    background-color: -apple-system-opaque-secondary-fill;
     font-family: system-ui;
-    color: -apple-system-blue;
     padding: 0.2em 0.5em;
+#if defined(WTF_PLATFORM_VISION) && WTF_PLATFORM_VISION
+    background-color: rgba(0, 0, 0, 0.04);
```

Comment:
This is a questionable choice (something like `color-mix(in srgb, CanvasText 4%, transparent)` would have been more accessible and given the same result), but this can be addressed separately.